src/index.js

```diff
@@ -4,41 +4,35 @@
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
 
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
-
-console.log('This is the JavaScript entry file - your code begins here.');
-
-console.log('proper clone test')
 
 import fetcher from './fetch.js';
+import User from './User.js';
 
-let userData, userTrips, tripsToDisplay, allDestinations;
+let userData, userTrips, tripsToDisplay, allDestinations, user;
 
 
 window.onload = fetchStuff;
-// window.onload = fetcher.fetchTrips
 window.addEventListener('click', clickLog);
 
 function fetchStuff() {
     fetcher.fetchUser(7);
-    // fetcher.fetchDestination()
-    // fetcher.fetchTripsForAUser()
-    // fetcher.fetchTrips().filter(trip => trip.userID === fetcher.fetchUser().id)
+    fetcher.fetchDestination();
 }
 
-let userStuff = {
+let fetchSetter = {
```

Comment:
The functionality for `fetchSetter` might be better off living in another class, because right now we have `fetcher` pulling in `index.js` functions to manipulate data.
These could potentially all be `user` functions. 
You can instantiate a user with data, like you're doing now, and have `user.setTrips(fetchedTrips)` and `user.setDestinations(fetchedDestinations)`

Reply:
This was insightful. I think I would like to use the User class to call these functionalities.